api/v1alpha1/virtualmachineservice_types.go

```diff
@@ -93,6 +93,13 @@ type VirtualMachineServiceSpec struct {
 	// +optional
 	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
 
+	// LoadBalancerSourceRanges is an array of IP addresses.
```

Comment:
I would think this would also take network notation in addition to individual ip addresses right? (eg. 10.0.0.0/16 type notation?)

Reply:
Good point! I updated the comment to make it clear that the string should be in the format of CIDRs.